steam.ly-content-service API.md

```diff
@@ -0,0 +1,26 @@
+# Valve Steam Content Service API
+
+Valve Steam is an online marketplace for video games.  The Content Service API provides access to all video games in the Valve Steam database.  The API uses RESTful calls and responses are formatted in JSON.
+
+## API Methods
+
+* /game/:id
+* /game
+
+### Arguments
```

Comment:
Are these `Arguments` for both the API methods above? It'd be good to be clear about this.

Reply:
I've now changed this, but /game/:id referred to a single game id, and /game referred to multiple game ids.  It's not easy to see that, I admit.